### ISSUE TITLE
neuron.py missing import dependency.

### ISSUE THREAD

**Issue by icsy7867** [camfairchild/bittensor#1405]
**Describe the bug**
Following the guide:
https://bittensor.com/documentation/mining/mining when trying to run neuron.py, even with the simple --help, You get a error:
NameError: name 'torch' is not define

**To Reproduce**
Follow the guide, try to use the neuron.py miner:
```
python3 neuron.py --help
```
get an error when running.

**Expected behavior**
It should work.

**Screenshots**
<img width="763" alt="image" src="https://github.com/opentensor/bittensor/assets/3788162/39132d67-d51c-4ef7-bcf2-7079386bc4db">

**Environment:**
 - vast.ai miner VM using for testing.
 - 
**Additional context**
Solution here appears to be fairly simple... I added:
```
import torch
```

to the top of the python file, and this runs as expected.

```
python3 ~/.bittensor/bittensor/neurons/text/prompting/miners/gpt4all/neuron.py --help
```

However, the documentation is incorrect on how to run this as well.  Following the guide:
```
python3 bittensor/neurons/text_prompting/miners/GPT4ALL/neuron.py
    --netuid SUBNETWORK_TARGET_UID
    --wallet.name YOUR_WALLET_NAME
    --wallet.hotkey YOUR_HOTKEY_NAME
    --logging.debug
```

However this produces an error:
```
neuron.py: error: the following arguments are required: --gpt4all.model
```

But I am having trouble finding any documentation what is required here, or what the other flags for gpt4all.x do, or what is required.

**Comment by camfairchild** [comment]
Hi, you can check out the [README](https://github.com/opentensor/bittensor/blob/95db4676bf364d1ce61e737eada884f8407fc3f1/neurons/text/prompting/miners/gpt4all/README.md) file for that miner for a guide on what flags to use. The usage is added there.

Also, the lines in the file [here](https://github.com/opentensor/bittensor/blob/fix/dont-finalize-set-weights/neurons/text/prompting/miners/gpt4all/neuron.py#L30) contain  the help strings for what the flags are for

**Comment by camfairchild** [comment]
Also fixed in #1460
